### PR TITLE
Configure hibernate naming strategy for liquibase

### PR DESCRIPTION
**Pull request by cbornet**

### CODE REVIEW

**Comment by cbornet** [comment]
This removes the need to set the name by JoinColumn annotations

**Comment by jdubois** [comment]
I'm trusting you on this (I didn't test), but yes it looks logical and a very good improvement!

File: entity/templates/src/main/java/package/domain/_Entity.java [hunk 1]
```diff
@@ -171,7 +170,7 @@
     <%_ } else { _%>
     <%_     if (relationships[relationshipId].ownerSide) { _%>
     @OneToOne
-    @JoinColumn(name = "<%= getColumnName(relationships[relationshipId].relationshipName) %>_id", unique = true)
```
Comment (deepu105) [inline entity/templates/src/main/java/package/domain/_Entity.java]: is this handled automatically by Hibernate as well?

Comment (cbornet) [inline entity/templates/src/main/java/package/domain/_Entity.java]: Yes,  in fact the PR is to make liquibase use the same naming strategy that we configure for spring in application.yml

Comment (deepu105) [inline entity/templates/src/main/java/package/domain/_Entity.java]: great :+1:

Thanks & Regards,
Deepu

On Wed, Jan 27, 2016 at 6:56 PM, Christophe Bornet <notifications@github.com

**Pull request merged**